src/libc/src/klee.c

```diff
@@ -0,0 +1,83 @@
+#include <assert.h>
+#include <klee.h>
+#include <owi.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+unsigned char *make_in(size_t size) {
+  unsigned char *t = malloc(size);
+  switch (size) {
+  case 8:
+    *t = owi_i8();
+    break;
+  case 32:
+    *t = owi_i32();
+    break;
+  case 64:
+    *t = owi_i64();
+    break;
+  default:
+    for (int i = 0; i < (size / sizeof(char)); i++) {
+      t[i] = owi_i8();
+    }
+    break;
+  }
+  return t;
+}
+
+void klee_define_fixed_object(void *addr, size_t nbytes) {
+  addr = make_in(nbytes);
```

Comment:
This is not doing what you expect. It should be `*addr = make_in(nbytes)`, to assign to the actual memory location pointed to by `addr`, otherwise, you are only modifying the value of the local `addr` variable.